css/datagrid/datagrid2.css

```diff
@@ -143,3 +152,30 @@
 .highcharts-dg-table thead th.highcharts-dg-hovered-column {
     background-color: #0000ff30;
 }
+
+.highcharts-dg-col-resizer {
+    position: absolute;
+    top: 0;
+    width: 10px;
+    right: 0;
+    height: 100%;
+    cursor: col-resize;
+    user-select: none;
+    z-index: 10;
+}
+
+.highcharts-dg-col-resizer::after {
+    content: "";
+    position: absolute;
+    display: block;
+    height: 20px;
+    width: 4px;
+    margin-top: -10px;
```

Comment:
?

Reply:
Like above, will probably be changed during the final styling of the DataGrid.